lms/templates/courseware/course_about.html

```diff
@@ -18,7 +18,10 @@
 </%block>
 
 <%block name="js_extra">
-
+  ## CourseTalk widget js script
+  % if show_coursetalk_widget:
+      <script src="//d3q6qq2zt8nhwv.cloudfront.net/s/js/widgets/coursetalk-read-reviews.js"></script>
```

Comment:
I'm not a fan of loading this JS from a CDN -- it's minified (so I can't review it), and it's completely decoupled from the platform which means future maintainers won't be able to change it if necessary. The code should be added into edx-platform (preferably as a RequireJS module). You can see examples of RequireJS code in a few different places in the platform. `lms/djangoapps/teams/static/teams/js` is a pretty good example of our current approach. We also have some docs about our usage of RequireJS [here on our wiki](https://openedx.atlassian.net/wiki/display/TNL/RequireJS).

Beyond that, I'd like to see tests for the JS component.